lib/order.rb

```diff
@@ -0,0 +1,20 @@
+class Order 
+    attr_reader :dishes, :total
+
+    def initialize(menuChoice_obj)
+        @dishes = menuChoice_obj
+        @total = 0
+    end
+
+    def sum
```

Comment:
maybe a separate class for sum will have been an idea to separate the responsibility, and the class order will maybe have encapsulate the menu, eventually basket for the selection of the dishes and the sum class for the total.